Dockerfile

```diff
@@ -29,7 +29,5 @@ RUN if [ "${ARCH}" != "s390x" ]; then \
 RUN install -s runc /usr/local/bin
 RUN runc --version
 
-FROM ubi
-RUN microdnf update -y && \
-    rm -rf /var/cache/yum
+FROM bci
```

Comment:
I noticed we're doing an update on the BCI image in other repos - is there any reason we're not doing that here as well?
```dockerfile
RUN zypper refresh && \
    zypper update -y && \
    zypper install -y gawk which && \
    zypper clean -a
```

Reply:
You're right, there's a mix of styles. It was intended to follow the existing approach, but it was overlooked here.

Reply:
Not a big deal, I was just curious if this might raise flags on image scanning since the packages won't get updated from those shipping on the base image.

As discussed on slack, nothing except the runc binary really matters since we just use this as an intermediate image for the rke2-runtime image. We could probably convert the final stage here and in the containerd and crictl images to `FROM scratch`.

Reply:
You're right, I missed that none of these images were being used for their runtimes at all. For the most part, the bci switch was a coarse way of resolving the existing image scan flags. Should this, containerd and crictl be switched to scratch before merging? Or should it be saved for later (also, there are some image-thinning opportunities for the other images that _do_ need a runtime)?